.github/workflows/ci.yaml

```diff
@@ -11,27 +11,24 @@ jobs:
     runs-on: ubuntu-latest
     strategy:
       matrix:
-        jvm: [hotspot, openj9]
         jvmType: [jdk, jre]
         version: [8, 11, 16]
     steps:
       - name: checkout
         uses: actions/checkout@v2
       - name: build
         env:
-          jvm: ${{ matrix.jvm }}
           jvmType: ${{ matrix.jvmType }}
           version: ${{ matrix.version }}
         run: |
-          cd "${jvm}/${jvmType}${version}"
-          docker pull "adoptopenjdk:${version}-${jvmType}-${jvm}"
-          docker build --tag "gradle:${jvmType}${version}-${jvm}" .
+          cd "${jvmType}${version}"
+          docker pull "adoptopenjdk:${version}-${jvmType}"
```

Comment:
Should this be temurin also?

Reply:
Yea. The jdk vs jre bit isn't right yet either.

I'm not committed to making this move yet. It seems like probably the best path forward, but there are some complications (see linked issue).